app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -126,6 +123,13 @@
     private ImageView arcPointer;
     private LinearLayout infoLayout;
 
+    private LinearLayout touchView;
+    private WindowManager.LayoutParams touchViewParams;
+    private Handler screenScanHandler;
+    private Runnable screenScanRunnable;
```

Comment:
I only meant final for the `screenScanRunnable`—it's just one instance. But it's a minor thing either way.